kitchenci/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+driver:
+  name: docker
+
+provisioner:
+  name: puppet_apply
+  puppet_version: "3.8.1"
```

Comment:
the puppet_version is already parametrized, I just wrote the latest available version as a default value, 
for driver value, yeah it worth parametrizing it, but this should be done from jenkins job configuration side with docker as a default value in .kitchen.yml file, make sense ?